scripts/getsamplesummary.py

```diff
@@ -1,90 +1 @@
-#!/usr/bin/python
-#
-import sys
-import datetime
-import time
-import glob
-import re
-import socket
-import os
-import select
-from access import db, lims
-
-outputdir = '/mnt/hds/proj/bioinfo/tmp/MIPP/exomes/'
-
-fc = "flowcell"
-if len(sys.argv) > 0:
-  try:
-    sys.argv[1]
-  except NameError:
-    sys.exit("Usage: " + sys.argv[0] + " <flowcell name>")
-  else:
-    fc = sys.argv[1]
-else:
-  sys.exit("Usage: " + sys.argv[0] + " <flowcell name>")
-
-params = db.readconfig("non")
-
-fc_samples = {}
-def getsamplesfromflowcell(pars, flwc):
-  samples = glob.glob(pars['DEMUXDIR'] + "*" + flwc + "*/Unaligned/Project_*/Sample_*")
-  for sampl in samples:
-    sample = sampl.split("/")[len(sampl.split("/"))-1].split("_")[1]
```

Comment:
This line could use a comment to explain what is going on.

Remembering from the walkthrough it chooses the basename of the path and extracts the second part of that string. This could be made clearer by using a builtin function in the `os` module:

```python
>>> import os
>>> os.path.basename(sampl)
D3-QXT-prep2
```